### PR TITLE
Update consume library for VS 2019

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #16046 

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/core/tutorials/consuming-library-with-visual-studio?branch=pr-en-us-16367)

### CODE REVIEW

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 1]
```diff
@@ -1,48 +1,47 @@
...
-
-* If the library will be generally accessible, you can distribute it as a NuGet package.
+- If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be publicly available, you can distribute it as a NuGet package.
 
 ## Include a library as a project in a solution
 
-Just as you included unit tests in the same solution as your class library, you can include your application as part of that solution. For example, you can use your class library in a console application that prompts the user to enter a string and reports whether its first character is uppercase:
+Just as you included unit tests in the same solution as your class library in [Test a .NET Standard library with .NET Core in Visual Studio 2017](testing-library-with-visual-studio.md), you can include your application as part of that solution. For example, you can use your class library in a console application that prompts the user to enter a string and reports whether its first character is uppercase:
```
Comment (tdykstra) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: Can we remove 2017 as VS version from the link text?

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 2]
```diff
@@ -1,48 +1,47 @@
...
 
 <!-- markdownlint-disable MD025 -->
 
 # [C#](#tab/csharp)
 
-1. Open the `ClassLibraryProjects` solution you created in the [Building a Class Library with .NET Standard in Visual Studio](library-with-visual-studio.md) topic. In **Solution Explorer**, right-click the **ClassLibraryProjects** solution and select **Add** > **New Project** from the context menu.
+1. Open the `ClassLibraryProjects` solution you created in the [Build a .NET Standard library in Visual Studio](library-with-visual-studio.md) article. In **Solution Explorer**, right-click the **ClassLibraryProjects** solution and select **Add** > **New Project** from the context menu.
```
Comment (tdykstra) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: article --> tutorial?

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 3]
```diff
@@ -66,27 +65,27 @@ Just as you included unit tests in the same solution as your class library, you
...
+   The program prompts the user to enter a string. It indicates whether the string starts with an uppercase character. If the user presses the Enter key without entering a string, the application ends, and the console window closes.
 
 1. If necessary, change the toolbar to compile the **Debug** release of the `ShowCase` project. Compile and run the program by selecting the green arrow on the **ShowCase** button.
 
    ![Debug on toolbar - Visual Basic](./media/consuming-library-with-visual-studio/visual-studio-project-toolbar.png)
 
 ---
 
-You can debug and publish the application that uses this library by following the steps in [Debugging your Hello World application with Visual Studio 2017](debugging-with-visual-studio.md) and [Publishing your Hello World Application with Visual Studio 2017](publishing-with-visual-studio.md).
+You can debug and publish the application that uses this library by following the steps in [Debugging your Hello World application with Visual Studio 2019](debugging-with-visual-studio.md) and [Publishing your Hello World Application with Visual Studio 2019](publishing-with-visual-studio.md).
```
Comment (tdykstra) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: Same question -- could we remove 2019 from link text?

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 4]
```diff
@@ -96,7 +95,7 @@ You can make your class library widely available by publishing it as a NuGet pac
 
    For example, enter **Command Prompt** in the search box on the Windows task bar. Select the **Command Prompt** desktop app or press **Enter** if it's already selected in the search results.
 
-1. Navigate to your library's project directory. Unless you've reconfigured the typical file location, it's in the *Documents\Visual Studio 2017\Projects\ClassLibraryProjects\StringLibrary* directory. The directory contains your source code and a project file, *StringLibrary.csproj*.
+1. Navigate to your library's project directory. Unless you've reconfigured the typical file location, it's in the *Documents\Visual Studio 2019\Projects\ClassLibraryProjects\StringLibrary* directory. The directory contains your source code and a project file, *StringLibrary.csproj*.
```
Comment (tdykstra) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: "reconfigured the typical location" sounds awkward, could we just say something like go to the folder where StringLIbrary.csproj is located?

**Review by tdykstra — APPROVED** [review]
LGTM, a couple suggestions.

File: docs/core/tutorials/consuming-library-with-visual-studio.md (hunk 1 above)
Comment (mairaw) [reply docs/core/tutorials/consuming-library-with-visual-studio.md]: yes. don't review this yet. I'm far from being done. Just wanted to get some builds started.

**Comment by mairaw** [comment]
@tdykstra @gewarren @BillWagner I think this one is ready for review. Let me know what you think of the style I've adopted without tabs. Given we've removed the initial screenshots, I think that is ok to just have the language selector. 

I could do the same for the [Build a .NET Standard class library in Visual Studio](https://review.docs.microsoft.com/en-us/dotnet/core/tutorials/library-with-visual-studio?branch=vs-2019&tabs=csharp) in a separate PR as well. I also noticed that the image for C# and VB look super different on that one (resolution, etc.). Not sure why.

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 5]
```diff
@@ -1,104 +1,83 @@
 ---
-title: Consume a .NET Standard library in Visual Studio 2017
-description: Build a .NET Core application that calls members of another class library with Visual Studio 2017.
+title: Consume a .NET Standard library in Visual Studio
+description: Build a .NET Core application that calls members of another class library with Visual Studio 2019.
 author: BillWagner
 ms.author: wiwagn
 ms.date: 06/05/2018
+dev_langs:
```
Comment (BillWagner) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: Should ms.date be updated?

**Review by BillWagner — APPROVED** [review]
This is great @mairaw 

You can :shipit: when ready.

I noted that you could / should update ms.date on the first article. That probably applies everywhere.

**Comment by tdykstra** [comment]
The problem with not using tabs is the language selector is hard to find.  For some reason the page defaults to VB for me, and if I just see "filename.cs or filename.vb" with VB code below it how would I know how to get C#? Would it be worth adding a note at the first occurrence of ".cs or .vb" pointing people to the selector?

File: docs/core/tutorials/consuming-library-with-visual-studio.md [hunk 6]
```diff
@@ -1,104 +1,83 @@
...
-* If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be used by a single solution (for example, if it's a component in a single large application), you can include it as a project in your solution.
+- If the library will be publicly available, you can distribute it as a NuGet package.
 
-* If the library will be generally accessible, you can distribute it as a NuGet package.
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+>
+> - Add a console app to your solution that references a .NET Standard library project.
+> - Create a NuGet package that contains a .NET Standard library project.
```
Comment (tdykstra) [inline docs/core/tutorials/consuming-library-with-visual-studio.md]: Given that the tutorial title says it's about consuming a library,  it seems like the "create a package" section should have instructions on how to consume the package as well.

**Review by tdykstra — APPROVED** [review]
One additional thought, but probably not for this PR.

**Comment by gewarren** [comment]
Personally I think the maintenance costs of tabs for C# and VB is too high to warrant it, so I like that you got rid of them. And hopefully someday soon the code language selector will have a better design, e.g. it would show up next to each code snippet.

**Comment by BillWagner** [comment]
> For some reason the page defaults to VB for me

@tdykstra I've found that is because you last looked at VB code (instead of C#). This is a UI concern that we've voiced a few times. I like your content suggestion to point to it on the first sample

File: docs/core/tutorials/consuming-library-with-visual-studio.md (hunk 5 above)
Comment (mairaw) [reply docs/core/tutorials/consuming-library-with-visual-studio.md]: Yes!

File: docs/core/tutorials/consuming-library-with-visual-studio.md (hunk 6 above)
Comment (mairaw) [reply docs/core/tutorials/consuming-library-with-visual-studio.md]: Good point @tdykstra. I might need to think of a better title. Some of the headings were also misleading. I think adding instructions could be an enhancement for later on beyond the scope of updating these topics for VS 2019.

Comment (mairaw) [reply docs/core/tutorials/consuming-library-with-visual-studio.md]: Opened a new issue to track this: https://github.com/dotnet/docs/issues/16369

**Pull request merged**